### ISSUE TITLE
Make study-time mebership require yearly renewal

### ISSUE THREAD

**Issue by DeD1rk** [svthalia/concrexit#2883]
### Is your feature request related to a problem? Please describe.
People don't email the board to notify them that they're no longer studying.

### Describe the solution you'd like
Instead of sending an email that they should notify, it would be nice if we can instead let them just click a button to confirm that they're still studying (i guess a checkbox 'i am still studying', and a confirm button to send the form). 

### Motivation
This way the member administration will be more accurate.

### Describe alternatives you've considered
(Only) adding a filter so board can get a list of study-time memberships to actively check who might no longer be studying.

### Additional context
Request by @CatoOri. It would be pretty awesome to have this at the start of next year.

**Comment by JobDoesburg** [comment]
> However, I'm not sure if we would get to see what for what study they're registered, so it might not give guarantees. 

I can assure you you will **not** see that information. You will just know that they are _a_ student

**Comment by JobDoesburg** [comment]
> And do you see added value in using saml authentication with s-number? (with a disclaimer that if they can't use it, they can contact the secretary?)

It would give some assurance on their name.

**Comment by DeD1rk** [comment]
> It would give some assurance on their name.

True, but on the other hand doing checks on that will involve us in the trouble uni has with getting internationals' names correctly

**Comment by DeD1rk** [comment]
We've decided not to implement a check with saml login for now

**Comment by CatoOri** [comment]
Looks good! Will it be possible to renew the membership through the website, like a yearly membership, for members that missed the email?

**Comment by DeD1rk** [comment]
Exactly, if they forget theur membership ends until they do it anyway
